app/scripts-browserify/search/index.js

```diff
@@ -94,14 +107,16 @@ function initialize() {
     if(updateWidgets) {
       if(config.features.filterSidebar) {
         const sidebar = require('./filter-sidebar');
-        // Update the sidebar right away
-        sidebar.update(searchParams.filters, null);
+        // Update the sidebar right away, as we're going of doing async work
+        // with searchParams, work on a clone.
+        const clonedParams = JSON.parse(JSON.stringify(searchParams));
```

Comment:
`clonedSearchParams` would be a better name, in my opinion, because it's more descriptive.

Reply:
:+1: